pkg/kv/kvserver/loqrecovery/loqrecoverypb/recovery.proto

```diff
@@ -51,11 +61,32 @@ message ReplicaInfo {
     (gogoproto.jsontag) = "raft_log_descriptor_changes,omitempty"];
 }
 
-// Collection of replica information gathered from a collect-info run on a single node.
+// Collection of replica information gathered in a collect-info run.
+// ReplicaInfo in Replicas does not have to be constrained to a single node,
+// but in practice info collected from remote cluster will contain info per
+// node. In case of offline collection, replicas will belong to all stores
+// provided to the command regardless of owning node.
 message NodeReplicaInfo {
```

Comment:
That would not work very well with offline collection where we are collecting all stores. There's no restrictions on it being a single node as the comment says. We were advertising this as a way to mount stores from multiple containers into single host and them running collection. I'm not comfortable of changing old behaviours in small ways straight after introducing them.

Reply:
I see. I suppose we could still group by node ID in those cases (each store must necessarily belong to one node), or otherwise explicitly mark the data as belonging to multiple nodes (e.g. with `node_id = 0`). But I'm also fine with deferring that until we actually need it -- for now, I think the only practical consequence is potential confusion/bugs for developers touching this code.